pkg/asset/internal/templates.go

```diff
@@ -729,15 +771,65 @@ spec:
       - name: ssl-certs-host
         hostPath:
           path: /usr/share/ca-certificates
-      - name: etc-kubernetes
-        hostPath:
-          path: /etc/kubernetes
+        name: ssl-certs-host
```

Comment:
accidental duplicate from a few lines above?

Reply:
yes, thanks for catching.